interactions/api/models/flags.py

```diff
@@ -94,6 +94,11 @@ class Permissions(IntFlag):
     MODERATE_MEMBERS = 1 << 40
 
 
+ALL_PERMISSIONS = Permissions(0)
+for perm in Permissions:
+    ALL_PERMISSIONS |= perm
+
+
```

Comment:
Is this an api thing?

Reply:
This is how it is documented in the [Discord pseudocode](https://discord.com/developers/docs/topics/permissions#permission-overwrites).
Ngl I've directly copied it from Molter ext, I had it written another way which is less beautiful